andela/ah-backend-iroquois
# #159952002 Send verification email on successful user registration.

#### What does this PR do?
Sends a user an email on successful signup

#### Description of the task to be completed
- Send user email on successful registration
- Embed a verification link in the email
- Activate user account

#### How should this be manually tested?
1.  Register a new user with the endpoint below in POSTMAN and set the method to "POST".
     http://127.0.0.1:8000/api/users/
{
	"user":{"username":"essa",
	"email":"essa@gmail.com",
	"password":"ghshgfhjasbdf"}
	
} 
2. Send the request. This request will send an email with the verification link in it.
3. Check your email for the verification link.
<img width="1098" alt="email" src="https://user-images.githubusercontent.com/22410594/45212955-e6b50500-b29e-11e8-9012-2fd775e0e5b9.png">
4. Copy the URL and test it in postman.
<img width="1144" alt="verified" src="https://user-images.githubusercontent.com/22410594/45213169-7064d280-b29f-11e8-9694-ad95fd4000cd.png">

5. The account will automatically be activated when you click that link.
6. Go ahead to log into your account.

#### What are the relevant pivotal tracker stories?
[#159952002](https://www.pivotaltracker.com/story/show/159952002)
